CMakeLists.txt

```diff
@@ -25,9 +25,6 @@ add_subdirectory(scripts)
 # This section contains variables that affect development GUIs (e.g. CMake GUI and IDEs), such as option(), folders, and variables
 # with the CACHE property.
 
-# Enable beta Vulkan extensions
-add_definitions(-DVK_ENABLE_BETA_EXTENSIONS)
```

Comment:
Should there be a reminder to re-enable it when the vulkan-hpp issues are resolved? I see that its specifically enabled for vulkaninfo *only*, which is fine by me. (Which assumes nothing else wants beta extensions enabled).

Reply:
Aside from fixing this particular issue, I think removing the global VK_ENABLE_BETA_EXTENSIONS is actually better long term. Individual targets like vulkaninfo and mockicd that want all that beta fun can opt in because they actually use it.